src/Nethermind/Nethermind.Init/Modules/WorldStateModule.cs

```diff
@@ -76,6 +77,7 @@ dbFactory is not MemDbFactory
 
             .Map<IWorldStateManager, PruningTrieStateFactoryOutput>((o) => o.WorldStateManager)
             .Map<IStateReader, IWorldStateManager>((m) => m.GlobalStateReader)
+            .Map<IPruningTrieStore, MainPruningTrieStoreFactory>((f) => f.PruningTrieStore)
```

Comment:
I think you dont need this anymore. Please dont use `IPruningTrieStore`. It will make migrating to other world state layout harder.